2-mandatory/2-financial-times.js

```diff
@@ -5,6 +5,13 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
+    let articlesWhichFit = []
+     for(item of allArticleTitles){
+        if (item.length < 65){
```

Comment:
what would happen if we have an article with exactly 65 characters in this case?

Reply:
@llamojha hi, ok, I understand)) will fix it to <= thanks!